test/fixes/issue-09.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var test = require('tape');
```

Comment:
i don't think we need a special folder just for "fixes"; this should just be part of normal parsing tests (`test/parse.js`)

Reply:
done